test/integration/targets/vmware_guest/tasks/clone_with_mac_addr.yml

```diff
@@ -0,0 +1,62 @@
+# Test code for the vmware_guest module.
+# Copyright: (c) 2017, James Tanner <tanner.jc@gmail.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+- name: Wait for Flask controller to come up online
```

Comment:
Hi,

We recently did a refactoring of the functional test-suite, the goal is to be able to run it on a regular VMware environment, this instead of vcsim.
As a consequence, you should not handle vcsim in the functional test anymore, use the `prepare_vmware_roles` role instead. e.g: https://github.com/ansible/ansible/blob/devel/test/integration/targets/vmware_drs_group/tasks/main.yml

You should also ensure the resources that are created during your test are properly removed at the end.